### PR TITLE
Update VSCode syntax highlighting for resource blocks.

### PR DESCRIPTION
**Pull request by samscott89**
Preview

image

### CODE REVIEW

File: vscode/oso/syntaxes/polar.tmLanguage.json [hunk 1]
```diff
@@ -2,48 +2,158 @@
...
-    { "include": "#inline-query" },
-    { "include": "#rule-def-head" },
-    { "include": "#rule-def-body" },
-    { "include": "#comment" }
+    {
+      "include": "#comment"
+    },
+    {
+      "include": "#fact"
+    },
```
Comment (samscott89) [inline vscode/oso/syntaxes/polar.tmLanguage.json:10]:
```suggestion
```

File: vscode/oso/syntaxes/polar.tmLanguage.json [hunk 2]
```diff
@@ -2,48 +2,158 @@
...
+    {
+      "include": "#rule"
+    },
+    {
+      "include": "#rule-type"
```
Comment (samscott89) [inline vscode/oso/syntaxes/polar.tmLanguage.json:12]: Crap, forgot to add this.

File: vscode/oso/package.json [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
+  "author": "Oso Security Inc.",
   "publisher": "osohq",
-  "displayName": "oso",
-  "description": "Syntax highlighting for .polar files as used by oso.",
+  "license": "Apache-2.0",
+  "displayName": "Oso",
+  "description": "Support for Oso policies written in Polar.",
   "icon": "oso_logo.png",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (saolsen) [inline vscode/oso/package.json:9]: maybe 0.2?

**Review by saolsen — APPROVED** [review]

File: vscode/oso/package.json (hunk 3 above)
Comment (samscott89) [reply vscode/oso/package.json:9]: Eh, this shouldn't be breaking. It should only be adding new highlights.

We'll save 0.2 for when it has LSP : )

**Pull request merged**